tests/reference/asr-test_assign8-4b26e63.stderr

```diff
@@ -1,5 +1,5 @@
-semantic error: readonly attribute
- --> tests/errors/test_assign8.py:4:5
+semantic error: Type mismatch in assignment, the types must be compatible
+ --> tests/errors/test_assign8.py:3:5
   |
-4 |     c.real = 5.0
-  |     ^^^^^^^^^^^^ 
+3 |     c = complex(3, 4)
+  |     ^   ^^^^^^^^^^^^^ type mismatch ('c32' and 'c64')
```

Comment:
This seems like a different error? I think you need to cast it first, so that you get to the actual error that we want to test.

Reply:
Fixed in [b5b4c82](https://github.com/lcompilers/lpython/pull/1271/commits/b5b4c82342721669d793585c84d3c87b922f53ae)